src/views/opm.tsx

```diff
@@ -535,199 +531,189 @@ const OPM: React.FC = () => {
-                                    title={form.label}
-                                    showTime={form.showTime}
-                                    iconPos={form.iconPos || "left"}
-                                    imgsrc={form.imgsrc}
-                                    onChange={(event) =>
-                                      handleFormChange(event)
-                                    }
-                                    value={form.value}
-                                    maxDate={
-                                      form.name === "startDate" ||
-                                      form.name === "endDate"
-                                        ? CURRENT_PST_DATE
-                                        : null
-                                    }
-                                  />
-                               
-                              )}
-                              {form.type === INPUT_TYPES.dropdown && (
-                               
-                                  <CustomDropdown
-                                    value={form.value}
-                                    name={form.name}
-                                    dropdownIcon={
-                                      <CustomImage src={ArrowDownIcon} />
-                                    }
-                                    onChange={(e) => handleFormChange(e)}
-                                   
-                                    imageclassname="top-3.5 z-1"
-                                    icon={form.icon}
-                                    options={form.options}
-                                    label={form.label}
-                                    optionLabel="name"
-                                    placeholder=""
-                                  />
-                               
-                              )}
-                            </>
-                          );
-                        })}
-                     
- 
-                      <CustomButton
-                      // id="page-btn-submit"
-                        label={LABELS.SUBMIT}
-                        isDisabled={disabled}
-                        isRounded={true}
-                        className="submitBtnMobile opmPopUp col-span-full"
-                      />
-                    </form>
-                  </CustomDialog>
-              </>
-            )}
-          </>
-        )
-      }
+              <CustomButton
+                id="page-btn-submit"
+                btnclassname="w-full"
+                label={LABELS.SUBMIT}
+                isDisabled={disabled}
+                isRounded={true}
+                onClick={submit}
+                className="self-end relative left-5w sm:w-21w md:w-15w lg:w-10w sm:top-2h md:top-0 sm:left-2.5w md:left-0 "
+              />
+            </>
+          ) : (
+            <>
+              <CustomDialog
+                header="Filters"
```

Comment:
Please move this to constant file if hardcoded.

Reply:
done